src/middleware.ts

```diff
@@ -16,6 +16,7 @@ export function createMiddleware(
       // Treat all relative URLs as the ones coming from the server.
       url: new URL(req.url, serverOrigin),
       headers: new Headers(req.headers as HeadersInit),
+      credentials: 'omit',
```

Comment:
https://github.com/mswjs/interceptors/blob/97a04bc6178bc3d20853cba5a4b559d9d4cdc494/src/createInterceptor.ts#L24-L29

> The value of the request client's "credentials" option or a compatible alternative (i.e. `withCredentials` for `XMLHttpRequest`). Always equals to "omit" in Node.js.

Reply:
A good catch. Thanks for adding this.